### PR TITLE
[Doc] Fix typos, anchor and examples

### PR DESCRIPTION
**Pull request by WiXSL**

### CODE REVIEW

**Review by slax57 — CHANGES_REQUESTED** [review]
Otherwise, as always, these are very welcome catches and improvements. Thank you so much!

File: docs/Actions.md [hunk 1]
```diff
@@ -273,7 +273,7 @@ const ApproveButton = () => {
     return <UpdateButton
         label="Approve"
         data={{ isApproved: true }}
-        mutationOptions={
+        mutationOptions={{
```
[the remark this answers is not in the document]
Comment (WiXSL) [reply docs/Actions.md:276]: It already has `{% raw %}` and `{% endraw %}` 😄

**Comment by WiXSL** [comment]
> Otherwise, as always, these are very welcome catches and improvements. Thank you so much!

You are welcome!

**Review by slax57 — APPROVED** [review]
Thanks!

**Pull request merged**